### PR TITLE
task/JavaScript Refreshment: task completed

### PR DESCRIPTION
**Pull request by JadeFox417**
- [ ] summary
- [ ] task 1: recreate the JavaScript code
- [ ] task 2: answer the question
- [ ] task 3: destructuring the JavaScript code
- [ ] task 4: change the (-) symbol into (/) symbol on the value
- [ ] task 5: do multiplication to the value
- [ ] task 6: round the value
- [ ] screenshots output
- [ ] essay

### CODE REVIEW

File: 08_Javascript-Refreshment/praktikum/task1.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var a = 5;
```
Comment (syukronarie) [inline 08_Javascript-Refreshment/praktikum/task1.js:1]: kenapa pake var?

File: 08_Javascript-Refreshment/praktikum/task1.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+let b = "kampus merdeka";
+const nama = "Budi";
+let terdaftar = false;
+let lengkap_arr = [a, b, nama, terdaftar];
```
Comment (syukronarie) [inline 08_Javascript-Refreshment/praktikum/task1.js:5]: variable di js pake camel case convention

Comment (JadeFox417) [reply 08_Javascript-Refreshment/praktikum/task1.js]: done mas, sudah saya perbaiki

File: 08_Javascript-Refreshment/praktikum/task1.js (hunk 1 above)
Comment (JadeFox417) [reply 08_Javascript-Refreshment/praktikum/task1.js]: sudah saya ganti let mas

**Review by syukronarie — APPROVED** [review]
reviewed and approved. tks

**Pull request merged**